apps/barista/src/components/color-grid/color-grid.ts

```diff
@@ -31,30 +31,44 @@ export class BaColorGrid implements AfterContentInit {
     this._colorInput = value;
   }
 
-  /** @internal name of the colorgroup */
-  _colorname: string;
+  @Input()
```

Comment:
Could you add a comment how this input should look like, i.e. how the colors have to be specified? :)